astropy/utils/compat/misc.py

```diff
@@ -7,14 +7,10 @@
 """
 
 import functools
-import sys
 
 from astropy.utils.decorators import deprecated
 
-__all__ = ["override__dir__", "PYTHON_LT_3_10", "PYTHON_LT_3_11"]
-
-PYTHON_LT_3_11 = sys.version_info < (3, 11)
-PYTHON_LT_3_10 = sys.version_info < (3, 10)
```

Comment:
the presence of `__all__` and the fact that this module contains a deprecation suggests that these are actually considered public API and shouldn't be removed without prior warning. Is that so ?

Reply:
We remove them when they aren't needed. How about you put them in sys.version_info

Reply:
Sorry I don't think I get what you're suggesting we do ?

Reply:
Yes, okay to just remove. `utils.compat` is for internal use.